include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -80,7 +85,7 @@ __enable_if_t<::std::is_integral<_T>::value && ::std::is_unsigned<_T>::value, _T
 __dpl_bit_floor(_T __x) noexcept
 {
     if (__x == 0) return 0;
-#if SYCL_LANGUAGE_VERSION >= 2020
+#if SYCL_LANGUAGE_VERSION
```

Comment:
It looks like `clz` was available in the spec for SYCL 1.2, so should we use `#if _ONEDPL_BACKEND_SYCL` to include `CL_SYCL_LANGUAGE_VERSION`?

Or can just assume it is available because according to the comment above:

`//!!! NOTE: This file should be included under the macro _ONEDPL_BACKEND_SYCL`

Reply:
You are almost correct :) Yes, clz is in SYCL 1.2.1, but it's `cl::sycl::clz` there. And we do not support SYCL 1.2.1 anyway.
We could assume that SYCL is available and remove all other versions of the code, but I think these utilities are really not specific to SYCL backend, and so I want to keep them more generic. Thus the TODO to move into the common header file (for now I just do not want to introduce many extra changes).

Reply:
Ah yes, since these are marked for movement to a common header file this works for me.  

I was just a little surprised by the `#if SYCL_LANGUAGE_VERSION` check as we do not use this specific check without an actual version number elsewhere in the repo.